### ISSUE TITLE
Sly binds C-c i and C-c x

### ISSUE THREAD

**Issue by uti5** [joaotavora/sly#515]
even though C-c [a-z] are supposed to be reserved for the user.

**Comment by joaotavora** [comment]
Yes, the `sly-package-fu` contrib does those. You can remove it from your `sly-contribs` list. I'm afraid these shortcuts are too historically useful (they were inherited from SLIME) to warrant removing.  Would break too many people's workflow.

Know any more?
